api/wasm/cpp/proxy_wasm_externs.h

```diff
@@ -164,6 +164,9 @@ extern "C" void proxy_on_grpc_trailing_metadata(uint32_t context_id, uint32_t to
 extern "C" void proxy_on_grpc_receive(uint32_t context_id, uint32_t token, uint32_t response_size);
 extern "C" void proxy_on_grpc_close(uint32_t context_id, uint32_t token, uint32_t status_code);
 
+extern "C" WasmResult proxy_active_span_set_tag(const char* key_ptr, size_t key_size, const char* value_ptr,
```

Comment:
At the ABI level, this should be called `proxy_set_active_span_tag()`.

Also, you should move it into host exports section above.

Reply:
Done.